scripts/common.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 ROOT=`dirname "$0"`
+unset RUSTC_WRAPPER
```

Comment:
What's the purpose of this? Could you add a comment?

Reply:
I was trying to figure out why `cargo run --dev` was failing, and I wondered if `sccache` was failing to recompile something it needed to.   I don’t think it was, though.